virttest/utils_libvirt/libvirt_migration.py

```diff
@@ -0,0 +1,93 @@
+"""
+libvirt related migration functions
+"""
+import os
+import logging
+
+from virttest import defaults
+from virttest import libvirt_vm
+from virttest import remote
+from virttest.utils_test import libvirt
+
+
+def update_virsh_migrate_extra_opts(params):
+    """
+    Update extra options for virsh migrate command
+
+    :param params: the parameters used
+    :return: the updated extra options
+    """
+    extra_opts = params.get("virsh_migrate_extra")
+    postcopy_options = params.get("postcopy_options")
+    if postcopy_options:
+        extra_opts = "%s %s" % (extra_opts, postcopy_options)
+    return extra_opts
+
+
+def update_virsh_migrate_extra_args(params):
```

Comment:
Move this to virttest/migration.py as a standalone function for now.

Reply:
updated it to 2979 as a method of MigrationTest since it generates the args for MigrationTest.do_migration.